sda/cmd/s3inbox/s3inbox.go

```diff
@@ -7,6 +7,8 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/gorilla/mux"
```

Comment:
```suggestion
```

MUX is part of the standard [HTTP library](https://pkg.go.dev/net/http@go1.22.2#ServeMux) since GO 1.22

Reply:
Yes, but to do route [HEAD  requests to /](https://github.com/neicnordic/sensitive-data-archive/pull/1025/files#diff-30d249c8dab4f2de2d776b274b7ea1cf0ae9bf858f21b88bfa8aa7f31099817dR107) I needed gorilla mux.

Reply:
with the standard library you should be able to do something like this:

```
mux.HandleFunc(GET "/health", proxy.CheckHealth ) // this will only accept GET and HEAD requests.
mux.HandleFunc("/", proxy)  // this will accept any request and then at the start of the proxy function where we detect which request it is we handle the HEAD request to "/" as one case.
```

But we can leave that for later since there are other things in this app that should be refactored.